browser/extensions/brave_wallet_navigation_throttle.cc

```diff
@@ -47,17 +48,20 @@ BraveWalletNavigationThrottle::WillStartRequest() {
       url.host() == ethereum_remote_client_host) {
     // If a user has explicitly disabled the Brave Wallet,
     // then don't defer and try to install it.
-    if (!Profile::FromBrowserContext(web_contents->GetBrowserContext())->
-        GetPrefs()->GetBoolean(kBraveWalletEnabled)) {
+    content::BrowserContext* browser_context =
+        web_contents->GetBrowserContext();
+    Profile* profile = Profile::FromBrowserContext(browser_context);
+    if (!Profile::FromBrowserContext(browser_context)->
+            GetPrefs()->GetBoolean(kBraveWalletEnabled) ||
```

Comment:
Did you mean to use the `profile` variable in the first part of this `if` statement too?

Reply:
might as well 👍